client/styles/abstracts/_variables.scss

```diff
@@ -1,4 +1,10 @@
 $base-font-size: 12;
+$sm: 576; // Small devices (landscape phones, 576px and up)
+$md: 768; // Medium devices (tablets, 768px and up)
+$lg: 992; // Large devices (desktops, 992px and up)
+$xl: 1200; // Extra large devices (large desktops, 1200px and up)
+$form-breakpoint: 390;
```

Comment:
rather than this being $form-breakpoint, i think it makes sense to have a number around this small that's the small device portrait number